http-clients/src/main/java/org/triplea/http/client/moderator/toolbox/ModeratorToolboxClient.java

```diff
@@ -10,13 +11,17 @@
 import org.triplea.http.client.HttpClient;
 import org.triplea.http.client.HttpInteractionException;
 
-import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Preconditions;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
 
 /**
  * Wrapper around moderator toolbox feign client. This wrapper handles exceptions and returns
  * a success string result for most methods, otherwise the return value is any error message
  * returned back from the server.
  */
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
 public class ModeratorToolboxClient {
```

Comment:
Updates here at to account for having a password now. This class is constructed with a password that is then used with any of the http endpoints. This class also knows how to pull the API key from client settings, with those two values any users of this class need not worry about key and password.